src/Test/Reporter/Highlightable.elm

```diff
@@ -0,0 +1,48 @@
+module Test.Reporter.Highlightable exposing (Highlightable, fromDiff, fromLists, map, resolve)
+
+import Diff exposing (Change(..))
+
+
+type Highlightable a
+    = Highlighted a
+    | Plain a
+
+
+resolve : { fromHighlighted : a -> b, fromPlain : a -> b } -> Highlightable a -> b
+resolve { fromHighlighted, fromPlain } highlightable =
+    case highlightable of
+        Highlighted val ->
+            fromHighlighted val
+
+        Plain val ->
+            fromPlain val
+
+
+fromLists : List a -> List a -> List (Highlightable a)
```

Comment:
It doesn't seem obvious that `Highlightable.fromLists` will do a `diff`. Is there a better name for this?

Reply:
👍 Renamed to`diffLists` in 5616fa615ad97f2a88341466115ccae67267b6bb